### PR TITLE
banner is being printed with a single logger call

### PR DESCRIPTION
**Pull request by pshirshov**
Fix for #1980

### CODE REVIEW

File: blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala [hunk 1]
```diff
@@ -278,7 +278,11 @@ class BlazeBuilder[F[_]](
         s"BlazeServer($address)"
     }
 
-    banner.foreach(logger.info(_))
+    Option(banner)
```
Comment (jmcardon) [inline blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala]: couldn't you just do this with a single `foldLeft` + an `isEmpty`?

Comment (pshirshov) [reply blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala]: I may though don't see much benefits in that. This is bit more verbose but looks more idiomatic (for my taste)

Comment (jmcardon) [reply blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala]: more of a code style thing, but if Ross doesn't mind I'll live with it.

Comment (rossabaker) [reply blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala]: If we start doing this as a single log statement, I don't think it needs to continue to be a `Seq`.  I think this works for 0.18 for bincompat, and we might simplify it on the server builders for 0.19.

**Pull request merged**